assets/airports/rjbb.json

```diff
@@ -0,0 +1,1958 @@
+            },
+            "exitPoints": {
+                "KRE": ["UMAJI", "KRE"],
+                "KEC": [["GOBOH", "A60+"], "KEC"]
+            },
+            "draw": [
+                ["TOMOH", "UMAJI", "KRE*"],
+                ["TOMOH", "GOBOH", "KEC*"]
+            ]
+        },
+        "KITAN1": {
+            "icao": "KITAN1",
+            "name": "Kitan One",
+            "rwy": {
+                "24L": ["BB440", "TOMOH"],
+                "24R": ["BB440", "TOMOH"]
+            },
+            "exitPoints": {
+                "TONDA": [["GOBOH", "A60+"], "TONDA"],
+                "YOSAK": [["ODORI", "A130+"], ["YOSAK", "A160+"]]
+            },
+            "draw": [
+                ["BB440", "TOMOH"],
+                ["GOBOH", "TONDA*"],
+                ["TOMOH", "ODORI", "YOSAK*"]
+            ]
+        },
+        "SUSAN1": {
+            "icao": "SUSAN1",
+            "name": "Susan One",
+            "rwy": {
+                "06L": ["BB430", "TOMOH"],
+                "06R": ["B6R10", ["B6R11", "A25+"], "B6R14", "BB430", "TOMOH"],
+                "24L": ["B4L30", "TOMOH"],
+                "24R": ["BB430", "TOMOH"]
+            },
+            "exitPoints": {
+                "TONDA": [["GOBOH", "A60+"], "TONDA"],
+                "YOSAK": [["ODORI", "A130+"], ["YOSAK", "A160+"]]
+            },
+            "draw": [
+                ["BB430", "TOMOH"],
+                ["B6R10", "B6R11", "B6R14", "BB430"],
+                ["B4L30", "TOMOH"],
+                ["TONDA*"],
+                ["YOSAK*"]
+            ]
+        },
+        "KNE1": {
+            "icao": "KNE1",
+            "name": "Kansai One",
+            "rwy": {
+                "06L": ["B6L20", ["KNE", "A80+"]],
+                "06R": ["B6R10", ["B6R11", "A25+"], "B6R13", ["KNE", "A80+"]],
+                "24L": ["B4L10", ["KNE", "A80+"]],
+                "24R": ["B4R20", ["KNE", "A80+"]]
+            },
+            "exitPoints": {
+                "NARAH": ["NARAH"],
+                "SHTLE": ["NARAH", "YAGYU", "SHTLE"],
+                "GUJYO": ["NARAH", "UENOH", "GUJYO"]
+            },
+            "draw": [
+                ["B6L20", "KNE"],
+                ["B6R11", "B6R13", "KNE"],
+                ["B4L10", "KNE"],
+                ["B4R20", "KNE"],
+                ["KNE", "NARAH*", "YAGYU", "SHTLE*"],
+                ["NARAH", "UENOH", "GUJYO*"]
+            ]
+        },
+        "NANKO1": {
+            "icao": "NANKO1",
+            "name": "Nanko One",
+            "rwy": {
+                "06L": ["B6L20", ["NANKO", "A80+"]],
+                "06R": ["B6R10", ["B6R11", "A25+"], "B6R13", ["NANKO", "A80+"]],
+                "24L": ["B4L10", ["NANKO", "A80+"]],
+                "24R": ["B4R20", ["NANKO", "A80+"]]
+            },
+            "exitPoints": {
+                "SIGAK": ["NANKO", "OGURA", "KYOTO", "SIGAK"],
+                "KCC": ["NANKO", "OGURA", "KYOTO", "SIGAK", "KCC"]
```

Comment:
in cases like this, where there are fixes that are present in _all_ the `exitPoints`, or all `rwy`s, for reasons of readability, please move them to a `body` section:
```
"rwy": {
    "06L": ["B6L20"],
    "06R": ["B6R10", ["B6R11", "A25+"], "B6R13"]
    "24L": ["B4L10"],
    "24R": ["B4R20"]
}
"body": [["NANKO", "A80+"], "OGURA", "KYOTO"],
"exitPoints": {
    "SIGAK": ["SIGAK"],
    "KCC": ["SIGAK", "KCC"]
}
```

You also do not want to have fixes listed in multiple sections, for example with `NANKO` in the `rwy` and `exitPoints` sections above. That will lead to there being two waypoints at the same place, and it's possible that could cause issues.